app/controllers/trainees_controller.rb

```diff
@@ -47,7 +47,11 @@ def create
   def update
     authorize trainee
     trainee.update!(trainee_params)
-    redirect_to page_tracker.last_origin_page_path
+    redirect_to trainee
+  end
+
+  def edit
```

Comment:
We need a guard here against editing a non-draft trainee. I can access the route manually in the URL with a non-draft trainee and update their `training_route` which I believe is a no-no!

cc. @edwardhorsford